test/button/button-subtle.visual-diff.js

```diff
@@ -21,6 +21,7 @@ describe('d2l-button-subtle', function() {
 	beforeEach(async function() {
 		await page.setViewport({width: 800, height: 800, deviceScaleFactor: 2});
```

Comment:
Did you determine if this line could be moved into the `before` to save some time?

Reply:
Ah yep... was able to move all of it which makes the run **much** faster.  Note: the durations that mocha reports is just for the test (`it` statement)... it does not include setup for each test.  So with these moved, the durations seem to reflect what I observe, as opposed to being this delay between each test.

Interestingly, with the `-s` threshold removed the delay due to waiting for the animation is now obvious.

![visual-diff-animation-time](https://user-images.githubusercontent.com/9042472/55658456-bc431000-57cb-11e9-8194-a70ab2b2b97f.png)